wordpress-mobile/WordPress-iOS
# Create Site: Create Site Epilogue

Fixes #8504 

To test:
- Go through the new site creation process.
- Create a site.
- The 'Congratulations' confetti view will appear for at least 4 seconds. After 4 seconds:
  - If the site is loaded, it will be displayed.
  - If the site is not loaded, the Congratulations view will continue to be displayed until it is.
- Once the site is loaded, it will fade in and the Congratulations view will fade out.
- The 'Configure' button dismisses the view and the new site's details will be shown.
- The 'Write first post' button dismisses the view and the post editor will be shown.

**Designed:**
![designed](https://user-images.githubusercontent.com/1816888/35459148-456c4986-029c-11e8-8a87-d76a36bb5678.png)

**Implemented:**
![implemented](https://user-images.githubusercontent.com/1816888/35464421-0eb1f3de-02b3-11e8-91b3-0fbc33e35cc4.png)

**Configure option selected:**
![configure](https://user-images.githubusercontent.com/1816888/35464562-ccd30a4c-02b3-11e8-9a9a-dbf9cce1af28.gif)

**Write Post option selected:**
![write_post](https://user-images.githubusercontent.com/1816888/35459204-6049b914-029c-11e8-8ba1-2790fa30f0cc.gif)
